src/R/Interpreters/Impl/IRInterpreterInfo.cs

```diff
@@ -33,6 +33,6 @@ public interface IRInterpreterInfo {
         /// </summary>
         /// <param name="fs"></param>
         /// <returns></returns>
-        bool VerifyInstallation(ISupportedRVersionRange svl = null, IFileSystem fs = null, ICoreShell coreShell = null);
+        bool VerifyInstallation(ISupportedRVersionRange svl = null, IFileSystem fs = null, IUIService ui = null);
```

Comment:
In this particular place it worries me because of the merge, cause I've split the `Microsoft.R.Interpreters` into crossplat and windows parts.